test/extended/node/node_e2e/node_utils.go

```diff
@@ -0,0 +1,55 @@
+package node
+
+import (
+	"strings"
+	"time"
+
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+func assertKubeletLogLevel(oc *exutil.CLI) {
+	var kubeservice string
+	var kublet string
+	var err error
+	waitErr := wait.Poll(10*time.Second, 1*time.Minute, func() (bool, error) {
+		nodeName, nodeErr := oc.AsAdmin().WithoutNamespace().Run("get").Args("nodes", "-o=jsonpath={.items[*].metadata.name}").Output()
```

Comment:
Assuming this code gets factored back in to the test, it would be good to see some By()s to clarify what the test is doing.